examples/mechanics/crack_branching.cpp

```diff
@@ -90,17 +90,18 @@ void crackBranchingExample( const std::string filename )
     double dy = particles->dx[1];
     double b0 = sigma0 / dy;
 
-    CabanaPD::RegionBoundary plane1( low_corner[0], high_corner[0],
-                                     low_corner[1] - dy, low_corner[1] + dy,
-                                     low_corner[2], high_corner[2] );
-    CabanaPD::RegionBoundary plane2( low_corner[0], high_corner[0],
-                                     high_corner[1] - dy, high_corner[1] + dy,
-                                     low_corner[2], high_corner[2] );
-    std::vector<CabanaPD::RegionBoundary> planes = { plane1, plane2 };
+    CabanaPD::RegionBoundary<CabanaPD::RectangularPrism> plane1(
+        low_corner[0], high_corner[0], low_corner[1] - dy, low_corner[1] + dy,
+        low_corner[2], high_corner[2] );
+    CabanaPD::RegionBoundary<CabanaPD::RectangularPrism> plane2(
+        low_corner[0], high_corner[0], high_corner[1] - dy, high_corner[1] + dy,
+        low_corner[2], high_corner[2] );
+    std::vector<CabanaPD::RegionBoundary<CabanaPD::RectangularPrism>> planes = {
```

Comment:
What happens if we could like to combine a RectangularPrism region with a different type of region?

Reply:
We would need a completely new option for that